src/main/java/com/example/school/model/Group.java

```diff
@@ -0,0 +1,19 @@
+package com.example.school.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+import java.util.UUID;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+public class Group {
+    private UUID id;
```

Comment:
For all `UUID id` you can also use `@Builder.Default` in this way: `private UUID id = randomUUID();`.